python/ray/scripts/scripts.py

```diff
@@ -221,8 +238,8 @@ def start(node_ip_address, redis_address, redis_port, num_redis_shards,
             object_store_memory=object_store_memory,
             num_workers=num_workers,
             cleanup=False,
-            redirect_worker_output=True,
-            redirect_output=True,
+            redirect_worker_output=not no_redirect_worker_output,
```

Comment:
The issue with not redirecting the worker output is that then the stdout/stderr on remote machines will be lost.

Reply:
They might be. But I might also be running `ray start` on my local machine, in a terminal, and observing stdout/stderr and I would like to see it in the terminal, not files.

Also, in my particular case, I will be intercepting stdout and stderr ourselves and collect those logs.

So, this allows me to configure Ray so that I can do that.

Reply:
And currently the default stays as it is. So by default it still redirects. PR adds a way to disable redirection to files.

Reply:
Note that it will appear in the terminal in which the `ray start` command was run, not the terminal in which your application was run. Is that the behavior you want?

Reply:
Yes. Because then that gets logged by the process supervisor.

And to collect information to the application I collect everything from Python logger into MongoDB.

Reply:
So I use a supervisor and I call it in block mode, and then not redirecting would mean that all output goes simply into one file by the supervisor.

But it is true that with latest updates to logging in Ray this whole PR not be necessary. So currently I manage to intercept logging and redirect it to MongoDB so I do not really care anymore if it is also stored into log files. It is unnecessary to log, but does not prevent things. Initially, this PR was made because I thought it will prevent me from collecting all the stdout.

So if you think this will complicate too much, we do not have to merge it. But I think it is useful to be able to control it in any case.

Reply:
Ok, seems relatively harmless and could help in some scenarios. As discussed, I think we'll have more comprehensive solutions in the future.